multiple-erc20-to-erc20/oracle/src/utils/utils.js

```diff
@@ -30,14 +30,14 @@ async function waitForFunds (web3, address, minimumBalance, cb, logger) {
     async retry => {
       logger.debug('Getting balance of validator account')
       const newBalance = web3.utils.toBN(await web3.eth.getBalance(address))
-      if (newBalance.gte(minimumBalance)) {
+      if (newBalance >= minimumBalance) {
```

Comment:
@Andrew-Pohl this was the cause of the error?
Reading the docs (and trying it out) both ways seems to be legit https://github.com/indutny/bn.js/

Reply:
Yes I saw the same but for me gte didn't seem to work new balance was 0 and that if was resolving as true...